### ISSUE TITLE
CRw: Process navigation, current step is not read as such

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#304]
![image](https://cloud.githubusercontent.com/assets/7510842/5219987/0a0c3404-765f-11e4-8120-05cc084ee1b3.png)

The current step of the process navigation is not read as "current" by the screenreader. See #216 
The step 4 should be read as "step 4, link not available*.
